main.tf

```diff
@@ -4,14 +4,14 @@ locals {
   image_initial_user = var.image_initial_user
   image_admin_group  = var.image_admin_group
 
-  server_id                  = var.server_id
-  server_name                = var.server_name
-  server_owner               = var.server_owner
-  server_type                = var.server_type
-  server_user                = var.server_user
-  server_ssh_key             = var.server_ssh_key
-  server_security_group_name = var.server_security_group_name
-  server_subnet_name         = var.server_subnet_name
+  server_id                  = var.id
```

Comment:
Do you want to lop off the server_ prefix on these like all of the others?

Reply:
so I am removing the server prefix user facing, but not internally
the idea was that it made sense to distinguish the image id from the server id internally, but not to the user

Reply:
the left side is a local var, the right is the interface var, I realized in a higher level module that the interface on this one stuck out because I was using the exact same variables in other modules without a prefix